tests/unit/color_picker.coffee

```diff
@@ -48,3 +48,16 @@ test 'Color picker converts color to hex when color is invalid', ->
   testHexConversion "foo", undefined
   testHexConversion "rgb(a, b, c)", undefined
   testHexConversion "rgb(1,2,3)", undefined
+
+test 'Custom selected color is set as custom color', ->
+  color = '#addec0'
+  colorPicker.set 'selectedColor', color
+  equal colorPicker.get('customColor'), color
+
+test 'Clicking color picker cell sets selectedColor', ->
```

Comment:
I don't think we should hold this fix up for tests since they haven't been running and seem to have some more fundamental issues that need to be addressed first.